### PR TITLE
Disable spotless groovy formatting

### PR DESCRIPTION
**Pull request by Vlatombe**
There was an outage in eclipse infrastructure today causing unnecessary build issues.

The plugin was accessing
https://download.eclipse.org/eclipse/updates/4.26/ which was returning a HTTP 502 error, and it was then throwing
`org.tukaani.xz.XZFormatException: Input is not in the XZ format` because it was downloading an html file with the http 502 error.

The spotless plugin should not bring dependencies from 3rd party websites that can't be mirrored in the jenkins infrastructure.

https://github.com/diffplug/spotless/issues/1697 is recording the problem on spotless side, but until then, getting stable builds outweights the benefit from formatting groovy files.

See [JENKINS-XXXXX](https://issues.jenkins.io/browse/JENKINS-XXXXX).

### Proposed changelog entries

- Entry 1: N/A
- […]

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] The Jira issue, if it exists, is well-described.
- [ ] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [ ] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

### CODE REVIEW

**Review by jglick — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]
/label ready-for-merge

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Comment by jglick** [comment]
Reverting enforcement part of #7834, FTR. Would solve https://issues.jenkins.io/browse/JENKINS-71147.

File: pom.xml
Comment (basil) [inline pom.xml:372]: Is the dependency on `shared-resources` a few lines below still needed now that `config/greclipse.properties` is no longer referenced by the configuration?

Comment (Vlatombe) [reply pom.xml:372]: Indeed, https://github.com/jenkinsci/jenkins/pull/8106

**Comment by jglick** [comment]
Looks like https://github.com/diffplug/spotless/pull/1821 could allow this to be enabled again in the future.

**Pull request merged**
